### DISCUSSION TITLE
iridescent effect

### DISCUSSION THREAD

**Discussion by DevIncredimate** [google/model-viewer#4052] [Q&A]
### Description

Hello, I have uploaded 3d Models to the MV and I want to give the "iridescent effect" to the glass Balls.
I know it is possible in Three.js 
How can we achieve that effect here? 

#### Live Demo

https://glitch.com/edit/#!/model-viewer

### Version

- model-viewer: v2.1.1

**Comment by elalish** [comment]
Yes, it's supported so just export your GLB with [iridescence extension](https://github.com/KhronosGroup/glTF/tree/main/extensions/2.0/Khronos/KHR_materials_iridescence) and it should work.
